models/public/mask_rcnn_inception_v2_coco/mask_rcnn_inception_v2_coco.md

```diff
@@ -0,0 +1,78 @@
+# mask_rcnn_inception_v2_coco
+
+## Use Case and High-Level Description
+
+Mask R-CNN Inception V2 trained on COCO dataset. Used for object instance segmentation. For details, see [paper](https://arxiv.org/pdf/1703.06870.pdf)
+
+## Example
+
+## Specification
+
+| Metric                          | Value                                     |
+|---------------------------------|-------------------------------------------|
+| Type                            | Instance segmentation                     |
+| GFlops                          | 54.926                                    |
+| MParams                         | 21.772                                    |
+| Source framework                | Tensorflow\*                              |
+
+## Performance
+
+## Input
+
+### Original Model
+
+Image, name: `image_tensor`, shape: [1x800x800x3], format: [BxHxWxC],
+   where:
+
+    - B - batch size
+    - H - image height
+    - W - image width
+    - C - number of channels
+
+   Expected color order: RGB.
+
+### Converted Model
+
+1. Image, name: `image_tensor`, shape: [1x3x800x800], format: [BxCxHxW],
+   where:
+
+    - B - batch size
+    - C - number of channels
+    - H - image height
+    - W - image width
+
+   Expected color order: BGR.
+
+2. Information of input image size, name: `image_info`, shape: [1x3], format: [BxC],
+   where:
+
+    - B - batch size
+    - C - vector of 3 values in format [H,W,S], where H is an image height, W is an image width, S is an image scale factor (usually 1)
+
+## Output
+
+### Original Model
+
+1. Classifier, name: `detection_classes`. Contains predicted bounding boxes classes in a range [1, 91]. The model was trained on the Microsoft\* COCO dataset version with 90 categories of objects, 0 class is for background.
+2. Probability, name: `detection_scores`. Contains probability of detected bounding boxes.
+3. Detection box, name: `detection_boxes`. Contains detection boxes coordinates in a format `[y_min, x_min, y_max, x_max]`, where (`x_min`, `y_min`)  are coordinates of the top left corner, (`x_max`, `y_max`) are coordinates of the right bottom corner. Coordinates are rescaled to input image size.
+4. Detections number, name: `num_detections`. Contains the number of predicted detection boxes.
+5. Segmentation mask, name: `detection_masks`. Contains segmentation masks of detected objects for each class.
+
+### Converted Model
+
+1. The array of summary detection information, name: `reshape_do_2d`, shape: [1, 1, N, 7], where N is the number of detected
```

Comment:
why [1, 1, N, 7]? `reshape_do_2d` should get [1, 1, N, 7] as input and return [N, 7] as output. max N is 100.